torch_xla/csrc/tensor.h

```diff
@@ -219,6 +219,14 @@ class XLATensor {
   static XLATensor threshold(const XLATensor& input, float threshold,
                              float value);
 
+  // Computes the ELU activation function, with scaling on input and output.
```

Comment:
These comment do not add much, do they?
I can see that's an ALU from the function name, and I can see the arguments.

Reply:
Done.